<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
	<head>
		<meta charset="utf-8" />
		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
				<title>Publications | GHERSILAB | Dario Ghersi</title>
		<meta name="author" content="Dario Ghersi" />
		<meta name="robots" content="all" />
		<meta name="generator" content="Sandvox 2.10.12" />
		<meta name="viewport" content="width=960" />
		<link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
		<link rel="canonical" href="http://fdf.com/publications.html" />
		
		<link rel="stylesheet" type="text/css" href="sandvox_Blueprint_modern/main.css" title="Blueprint Modern" /><!--[if IE 6]>
		<link rel="stylesheet" type="text/css" href="sandvox_Blueprint_modern/ie6.css" /><![endif]--><!--[if IE 7]>
		<link rel="stylesheet" type="text/css" href="sandvox_Blueprint_modern/ie7.css" /><![endif]-->
		<!--
		Photo credits for this website's design: <http://fdf.com/sandvox_Blueprint_modern/Credits.rtf>
		Licensing for this website's design:     <http://fdf.com/sandvox_Blueprint_modern/License.rtf>
		-->
		
	</head>
	<body class="sandvox has-page-title allow-sidebar no-custom-banner no-navigation no-IR" id="fdf_com" >
				<div id="page-container">
			<div id="page">
				<div id="page-top" class="has-logo has-title has-tagline">
					<div id="title">
						<a href="./" class="imageLink"><span id="logo-container"><img id="logo" src="Media/file34890_med_hr.jpeg" alt="file34890" width="200" height="109" /></span></a>
						<h1 class="title in"><a href="./"><span class="in">GHERSILAB</span></a></h1>
						<p id="tagline"><span class="in">computational biology @ <a href="http://bioinformatics.ist.unomaha.edu/index.php" target="_blank">University of nebraska at omaha</a></span></p>
					</div><!-- title -->
					<div id="sitemenu-container">
						<div id="sitemenu">
							<h2 class="hidden">Site Navigation<a href="#page-content" rel="nofollow">[Skip]</a></h2>
							<div id="sitemenu-content">
								<ul>
									<li class="i1 o"><a href="./" title="Research interests"><span class="in">Research Interests</span></a></li>
									<li class="i2 e currentPage"><span class="in">Publications</span></li>
									<li class="i3 o"><a href="software.html" title="Software"><span class="in">Software</span></a></li>
									<li class="i4 e"><a href="group.html" title="Group"><span class="in">Group</span></a></li>
									<li class="i5 o"><a href="teaching.html" title="Teaching"><span class="in">Teaching</span></a></li>
									<li class="i6 e last-item last"><a href="where-we-are.html" title="Where we are"><span class="in">Contact</span></a></li>
								</ul>
							</div> <!-- /sitemenu-content -->
						</div> <!-- /sitemenu -->
					</div> <!-- sitemenu-container -->
				</div> <!-- page-top -->
				<div class="clear below-page-top"></div>
				<div id="page-content" class="no-navigation">
					<div id="sidebar-container">
						<div id="sidebar">
							<div id="sidebar-top"></div>
							<div id="sidebar-content">
								<h3 class="hidden">Sidebar<a rel="nofollow" href="#main">[Skip]</a></h3>
								<div class="pagelet titled i1 o">
									<h4 class="title pagelet-title"><span class="in">Google Scholar profile</span></h4>
									<div class="pagelet-body">
										<div class="RichTextElement">
											<div>
												<p><a href="http://scholar.google.com/citations?user=WZ6huJ0AAAAJ&amp;hl=en&amp;oi=ao" target="_blank">http://scholar.google.com/citations?user=WZ6huJ0AAAAJ&amp;hl=en&amp;oi=ao</a></p>
												
											</div>
										</div>
									</div>
								</div>
								<div class="pagelet untitled i2 e last-item">
									<div class="pagelet-body">
										<div class="ImageElement">
											<div class="photo">
												<div>
													<!-- sandvox.ImageElement --><img src="Media/images-1_med_hr.jpeg" alt="images-1" width="228" height="129" />
													<!-- /sandvox.ImageElement --></div>
											</div>
										</div>
									</div>
								</div>
							</div> <!-- sidebar-content -->
							<div id="sidebar-bottom"></div>
						</div> <!-- sidebar -->
					</div> <!-- sidebar-container -->
					<div id="main">
						<div id="main-top"></div>
						<div id="main-content">
							<h2 class="title"><span class="in">Publications</span></h2>
							<div class="article">
								<div class="article-content">
									<div class="RichTextElement">
										<div><p><strong style="font-family: 'Helvetica Neue', Arial, Helvetica, Geneva, sans-serif; font-size: 1em; font-style: normal; font-variant: normal;">2018</strong></p><ul><li><span style="font-family: Times; font-size: 13px; font-style: normal; font-variant: normal; color: rgb(0, 0, 0);">Munro D, </span><u style="font-family: Times; font-size: 13px; font-style: normal; font-variant: normal; color: rgb(0, 0, 0);">Ghersi D</u><span style="font-size: 13px; font-family: Times; color: rgb(0, 0, 0);">, Singh M, </span><span style="font-size: 13px; font-family: Times; color: rgb(0, 0, 0);">“</span><span style="font-size: 13px; font-family: Times; color: rgb(0, 0, 0);">Two critical positions in zinc finger domains are heavily mutated in three human cancer types</span><span style="font-size: 13px; font-family: Times; color: rgb(0, 0, 0);">”</span><span style="font-size: 13px; font-family: Times; color: rgb(0, 0, 0);">, <em>PLOS Computational Biology </em>2018</span></li></ul><p><strong style="font-family: 'Helvetica Neue', Arial, Helvetica, Geneva, sans-serif; font-size: 1em; font-style: normal; font-variant: normal;">2017</strong></p><ul><li><span style="font-size: 13px; font-family: Times; color: rgb(0, 0, 0);"><u>Ghersi D</u>, Parakh A, Mezei M, “Comparison of a quantum random number generator with pseudorandom number generators for their use in molecular Monte Carlo Simulations”, <em>Journal of Computational Chemistry</em>, in press.</span></li><li><span style="font-size: 13px; font-family: Times; color: rgb(0, 0, 0);">Aslan N, Watkin L, Gil A, Mishra R, Clark F, Welsh R, <u>Ghersi D,</u> Luzuriaga K, Selin LK, </span><span style="font-size: 13px; font-family: Times; color: rgb(0, 0, 0);">“</span><span style="font-size: 13px; font-family: Times; color: rgb(0, 0, 0);">Severity of acute infectious mononucleosis correlates with cross-reactive influenza CD8 T-cell receptor repertoires</span><span style="font-size: 13px; font-family: Times; color: rgb(0, 0, 0);">”</span><span style="font-size: 13px; font-family: Times; color: rgb(0, 0, 0);">, <em>mBio</em> 2017, 10.1128/mBio.01841-17</span></li><li><span style="font-family: Times; font-size: 13px; color: rgb(0, 0, 0);">Watkin LB, Mishra R, Gil A, Aslan N, Ghersi D, Luzuriaga K, Selin LK, “Unique influenza A cross-reactive T-cell receptor repertoire has a potential to protect against EBV seroconversion”, <em>Journal of Allergy and Clinical Immunology 2017, 140(4): 1206-1210</em></span></li><li><span style="font-size: 13px; font-family: Times; color: rgb(0, 0, 0);">Young IY, Gil A, Mishra R, </span><u style="color: rgb(0, 0, 0); font-family: Times; font-size: 13px; font-style: normal; font-variant: normal;">Ghersi D</u><span style="font-size: 13px; font-family: Times; color: rgb(0, 0, 0);">, Selin LK, Stern LJ, “Broad TCR repertoire and diverse structural solutions for recognition of an immunodominant CD8+ T cell epitope”, <em>Nature Structural and Molecular Biology</em> 2017, 10.1038/nsmb.3383</span></li></ul><p><strong style="font-family: 'Helvetica Neue', Arial, Helvetica, Geneva, sans-serif; font-size: 1em; font-style: normal; font-variant: normal;">2016</strong></p><ul><li><span style="font-size: 13px; font-family: Times; color: rgb(0, 0, 0);">McGrath S, </span><u style="color: rgb(0, 0, 0); font-family: Times; font-size: 13px; font-style: normal; font-variant: normal;">Ghersi D</u><span style="font-size: 13px; font-family: Times; color: rgb(0, 0, 0);">, “Building towards precision medicine: empowering medical professionals for the next revolution”, <em>BMC Medical Genomics </em>2016<em>,</em> 9(1), 10.1186/s12920-016-0183-8</span><span style="color: rgb(0, 0, 0);"><u><br /></u></span></li><li><span style="font-size: 13px; font-family: Times; color: rgb(0, 0, 0);">Sharma S, Karri K, Thapa I, Bastola D, </span><u style="color: rgb(0, 0, 0); font-family: Times; font-size: 13px;">Ghersi D</u><span style="font-size: 13px; font-family: Times; color: rgb(0, 0, 0);">, “Identifying enriched drug fragments as possible candidates for metabolic engineering”, <em>BMC Medical Genomics </em>2016, 9(2): 2-46</span></li><li><span style="font-size: 13px; font-family: Times; color: rgb(0, 0, 0);"><u>Ghersi D</u>, Sanchez R, “Recovering bound forms of protein structures using the Elastic Network Model and Molecular Interaction Fields”, <em>Proceedings of the ACM-BCB 2016 Conference</em></span></li></ul><p style="font-weight: 300;"><strong style="font-size: 1em;">2015</strong></p><ul><li><span style="color: rgb(0, 0, 0); font-size: 13px; font-family: Times;">Balasubramanya A, Thapa I, Bastola D, <u>Ghersi D</u>, </span><span style="font-size: 13px; font-family: Times; color: rgb(0, 0, 0);">“A novel approach to identify shared fragments in drugs and natural products”, <em>2015 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</em></span></li><li><span style="color: rgb(0, 0, 0); font-size: 13px; font-family: Times;">Pritykin Y, </span><span style="font-size: 13px; font-family: Times;"><u style="color: rgb(0, 0, 0);">Ghersi D</u></span><span style="color: rgb(0, 0, 0); font-size: 13px; font-family: Times;">, Singh M, “Genome-Wide Detection and Analysis of Multifunctional Genes”, <em>PLoS Computational Biology </em>2015, 10.1371/journal.pcbi.1004467</span></li></ul><p style="font-weight: 300;"><strong style="font-size: 1em;">2014</strong></p><ul><li><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;"><u>Ghersi D</u>, Singh M, “molBLOCKS: a suite to break down small molecules into fragments and analyze
																																																																																																																																																																																						their distribution”, </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMTI10;"><em>Bioinformatics</em> </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">2014, 10.1093/bioinformatics/btu173</span></li></ul><p><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;"><strong>2013</strong></span></p><ul><li><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;"><u>Ghersi D</u>, Singh M, “Interaction-based discovery of functionally important genes in cancers”, </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMTI10;"><em>Nucleic
																																																																																																																																																																																						Acids Research</em> </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">2013, 10.1093/nar/gkt1305</span></li><li><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;"><u>Ghersi D</u>, Singh M, “Disentangling the complex relationship between function, network features and
																																																																																																																																																																																						disease: an application to cancer and Mendelian disorders”, </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMTI10;"><em>BMC Systems Biology</em> </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">2013, 7:5</span></li></ul><p><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;"><strong>2012</strong></span></p><ul><li><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;"><u>Ghersi D</u>, Sanchez R, “Automated Identification of Binding Sites for Phosphorylated Ligands in
																																																																																																																																																																																						Protein Structures”, </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMTI10;"><em>Proteins</em> </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">2012, 80:2347-58</span></li></ul><p><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;"><strong>2011</strong></span></p><ul><li><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;"><u>Ghersi D</u>, Sanchez R, “Beyond structural genomics: computational approaches for the identification
																																																																																																																																																																																						of ligand binding sites in protein structures”, </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMTI10;"><em>Journal of Structural and Functional Genomics</em> </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">2011,
																																																																																																																																																																																						12(2):109</span></li><li><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">Chakravarty S, <u>Ghersi D</u>, Sanchez R, “Systematic Assessment of Accuracy of Comparative Model
																																																																																																																																																																																						of Proteins Belonging to Different Structural Fold Classes”, </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMTI10;"><em>J Mol Mod</em> </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">2011, 17(11):2831</span></li><li><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">Calcagno C, Puzone R, Pearson YE, Cheng Y, <u>Ghersi D</u>, Selin LK, Welsh RM, Celada F “Computer
																																																																																																																																																																																						simulations of heterologous immunity: highlights of an interdisciplinary cooperation”, </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMTI10;"><em>Autoimmunity</em>
																																																																																																																																																																																						</span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">2011, 44(4):304</span></li></ul><p><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;"><strong>2010</strong></span></p><ul><li><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">Zhang Q, Chakravarty S, <u>Ghersi D</u>, Plotnikov AN, Sanchez R, Zhou MM, “Biochemical Profiling of
																																																																																																																																																																																						Histone Binding Selectivity of the Yeast Bromodomain Family”, </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMTI10;"><em>PLoS One</em> </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">2010, 5(1):e8903</span></li></ul><p><span style="font-size: 13px; font-family: CMR10;"><strong>2009</strong></span></p><ul><li><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;"><u>Ghersi D</u>, Sanchez R, “</span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMTT10;">EasyMIFs </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">and </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMTT10;">SiteHound</span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">: tools for the identification of ligand-binding sites
																																																																																																																																																																																						in protein structures”, </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMTI10;"><em>Bioinformatics</em> </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">2009, 25(23):3185-6</span></li><li><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">Hernandez M, <u>Ghersi D</u>, Sanchez R, “SITEHOUND-web: a server for ligand binding site identifica-
																																																																																																																																																																											tion in protein structures”, </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMTI10;"><em>Nucleic Acids Research</em> </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">2009, 37: W413-W416 </span></li><li><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;"><u>Ghersi D</u>, Sanchez R, “Improving accuracy and efficiency of blind protein-ligand docking by focusing
																																																																																																																																																																																						on predicted binding sites”, </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMTI10;"><em>Proteins</em> </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">2009, 4(2):417-24</span></li><li><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">Cheng Y, <u>Ghersi D</u>, Calcagno C, Selin LK, Puzone R, Celada F, “A discrete computer model of
																																																																																																																																																																											the immune system reveals competitive interactions between the humoral and cellular branch and
																																																																																																																																																																											between cross-reacting memory and naive responses”, </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMTI10;"><em>Vaccine</em> </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">2009, 27(6):833-45</span></li></ul><p><span style="font-size: 13px; font-family: CMR10;"><strong>2006</strong></span></p><ul><li><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">Bahl K, Kim SK, Calcagno C, <u>Ghersi D</u>, Puzone R, Celada F, Selin LK, Welsh RM “IFN-induced
																																																																																																																																																																											attrition of CD8 T cells in the presence or absence of cognate antigen during the early stages of viral
																																																																																																																																																																											infections”, </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMTI10;"><em>J Immunol</em> </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">2006, 176(7):4284-95</span></li><li><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">Cornberg M, Chen AT, Wilkinson LA, Brehm MA, Kim SK, Calcagno C, <u>Ghersi D</u>, Puzone R,
																																																																																																																																																																											Celada F, Welsh RM, Selin LK “Narrowed TCR repertoire and viral escape as a consequence of
																																																																																																																																																																											heterologous immunity”, </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMTI10;"><em>J Clin Invest</em> </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">2006, 116(5):1443-56</span></li></ul><p><span style="font-size: 13px; font-family: CMR10;"><strong>2004</strong></span></p><ul><li><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">Selin LK, Cornberg M, Brehm MA, Kim SK, Calcagno C, <u>Ghersi D</u>, Puzone R, Celada F, Welsh
																																																																																																																																																																											RM ”CD8 memory T cells: cross-reactivity and heterologous immunity“, </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMTI10;"><em>Semin Immunol</em> </span><span style="color: rgb(0, 0, 0); font-size: 10pt; font-family: CMR10;">2004
																																																																																																																																																																											16(5):335-47 </span></li></ul>
											</div>
									</div>
								</div> <!-- /article-content -->
								<div class="article-info">
								</div> <!-- /article-info -->
							</div> <!-- /article -->
						</div> <!-- main-content -->
						<div id="main-bottom"></div>
					</div> <!-- main -->
				</div> <!-- content -->
				<div class="clear below-content"></div>
				<div id="page-bottom">
					<div id="page-bottom-contents">
						<div>© Dario Ghersi 2017</div>
						<div class="hidden"> <a rel="nofollow" href="#title">[Back To Top]</a></div>
					</div>
				</div> <!-- page-bottom -->
			</div> <!-- container -->
			<div id="extraDiv1"><span></span></div><div id="extraDiv2"><span></span></div><div id="extraDiv3"><span></span></div><div id="extraDiv4"><span></span></div><div id="extraDiv5"><span></span></div><div id="extraDiv6"><span></span></div>
		</div> <!-- specific body type -->
		<script src="//ajax.aspnetcdn.com/ajax/jQuery/jquery-1.9.1.min.js"></script>
		<script>
		if (typeof jQuery === 'undefined') document.write('<scr'+'ipt src="Resources/jquery-1.9.1.min.js"></scr'+'ipt>');
		</script>
		<script charset="utf-8" src="sandvox_Blueprint_modern/javascript.js"></script>
		
	</body>
</html>
